settings.py

```diff
@@ -752,7 +752,7 @@ def _color_scheme_completions(view):
                 "{}  \tPackage: {}".format(file_name, package),
                 scheme_path
             )
-            completions.add(item)
+            completions.append(item)
         return completions
```

Comment:
Your changes in value_completions() broke a couple of things. 😠 

- automatic adding/stripping quotes to/from completions is broken,
- completions for float values (e.g. "dpi_scale") are broken
- some completions are sorted several times

somehow much more error prone than before